api/library/EventSource/Store.hs

```diff
@@ -234,6 +263,54 @@ foldEvents store stream k seed =
   foldEventsM store stream (\s a -> return $ k s a) seed
 
 --------------------------------------------------------------------------------
+-- | Like `forEvents` but provides access to SavedEvents instead of
```

Comment:
`s/SavedEvents/'SavedEvent's` That way, on documentation generation, it will link to `SavedEvent` type declaration.